examples/deployAccount/main.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/NethermindEth/juno/core/felt"
+	starknetgo "github.com/NethermindEth/starknet.go"
+	"github.com/NethermindEth/starknet.go/rpc"
+	"github.com/NethermindEth/starknet.go/utils"
+	ethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/joho/godotenv"
+)
+
+var (
+	network              string = "testnet"
+	predeployedClassHash        = "0x2794ce20e5f2ff0d40e632cb53845b9f4e526ebd8471983f7dbd355b721d5a"
+)
+
+func main() {
+	// Initialise the client.
+	godotenv.Load(fmt.Sprintf(".env.%s", network))
+	base := os.Getenv("INTEGRATION_BASE")
+	c, err := ethrpc.DialContext(context.Background(), base)
+	if err != nil {
+		panic("You need to specify the testnet url in .env.testnet")
+	}
+	clientv02 := rpc.NewProvider(c)
+
+	// Get keys
+	pub, priv := getRandomKeys()
+
+	classHash, err := utils.HexToFelt(predeployedClassHash)
+	if err != nil {
+		panic(err)
+	}
+
+	tx := rpc.BroadcastedDeployAccountTransaction{
```

Comment:
This type was removed in a previous PR, I guess it should be `DeployAccountTxn` instead